airflow/www/views.py

```diff
@@ -31,7 +31,16 @@
 from collections import defaultdict
 from functools import cached_property, wraps
 from json import JSONDecodeError
-from typing import TYPE_CHECKING, Any, Callable, Collection, Iterator, Mapping, MutableMapping, Sequence
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Collection,
+    Iterator,
+    Mapping,
+    MutableMapping,
+    Sequence,
+)
```

Comment:
This could be related to a change which you reverted
```suggestion
from typing import TYPE_CHECKING, Any, Callable, Collection, Iterator, Mapping, MutableMapping, Sequence
```